app/templates/buyers/brief_overview.html

```diff
@@ -38,8 +38,23 @@
           </form>
         </div>
       {% endif %}
-   {% endblock %}
+
+      {% with messages = get_flashed_messages(with_categories=true) %}
+        {% if messages %}
+          {% for category, message in messages %}
+            {%
+            with
+            message = "You've updated {}".format(message.get('updated-brief')) if 'updated-brief' in message else message,
```

Comment:
Can we get a unit test for this please. You could likely do without the one for the error state but the positive state should be covered.

Reply:
Done 👍